### ISSUE TITLE
AreOpenCloseAnimationsEnabled="false" in MenuFlyout not working

### ISSUE THREAD

**Issue by idrisnacer** [Kinnara/ModernWpf#80]
Hi,
I'm trying to disable MenuFlyout animation with :
`AreOpenCloseAnimationsEnabled="false"`
but it didn't work

**version  :** 0.8.3
**dotneetcore version :** 3.1.101

**Comment by Kinnara** [comment]
Thanks for reporting. A fix has been committed and will be included in the next 0.9 release. As a workaround for now, you can use the following code:
```xaml
<UserControl.Resources>
    <PopupAnimation x:Key="{x:Static SystemParameters.MenuPopupAnimationKey}">None</PopupAnimation>
</UserControl.Resources>
```

**Comment by idrisnacer** [comment]
Great, thank you 😄 
I'm using the workaround for now, and it's working 👍
